examples/locality-aware-routing/README.md

```diff
@@ -0,0 +1,143 @@
+# Locality based routing between ECS tasks
+
+This example demonstrates how Consul routes traffic based on the locality where the ECS tasks are deployed. As of 1.17, Consul only supports locality aware routing within a single partition. Support for multiple partitions and multiple cluster peers will soon be added in the upcoming releases.
+
+**Note**: This feature only works for a Consul enterprise installation setup.
+
+![Example architecture](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/locality-aware-routing-arch.png?raw=true)
+
+This terraform example deploys a client application and two server application tasks spread across available within the same AWS region. The consul server is also deployed as an ECS task along with an ECS controller.
+
+## Requirements
+
+* `jq`
+* `curl`
+* Terraform >= 1.2.2
+* Authentication credentials for the [Terraform AWS provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication)
+
+## Usage
+
+### Setup
+
+Clone this repository:
+
+```console
+$ git clone https://github.com/hashicorp/terraform-aws-consul-ecs.git
+$ git checkout tags/<latest-version>
+$ cd terraform-aws-consul-ecs/examples/locality-aware-routing
+```
+
+This module contains everything needed to spin up the example. The only
+requirements are:
+- You need to pass in the `name` variable. This value will be used as a unique identifier
+  for all resources created by the example. The examples below use the name `ecs`.
+- You need to pass in the IP address of your workstation via the `lb_ingress_ip`
+  variable. This is used for the security groups on the elastic load balancers to ensure
+  only you have access to them.
+- Consul Enterprise license that needs to be passed to the `consul_license` variable. This license will be used to run the enterprise version of Consul in this example. One way to pass these licenses would be add them to a `.tfvars` file and pass it as an argument to the `terraform apply` command.
+
+Determine your public IP. You can use a site like https://ifconfig.me/:
+
+```console
+$ curl ifconfig.me
+123.456.789.1%
+```
```

Comment:
strangely, `curl ifconfig.me` no longer works for me. I get `301 Moved Permanently` :thinking: 

I had to use `curl -s https://ipinfo.io/ip` instead. Do you get the same?

When I browse to `ifconfig.me` I see an IPv6 address, so I'm wondering if that has something to do with it.

Reply:
Nope.`curl ifconfig.me` works fine for me

Reply:
OK, I'm resolving this then. It is likely a problem on my end.